roles/common/defaults/main.yml

```diff
@@ -13,7 +13,12 @@ ats_home: "{{ binaries_folder }}/transform-service"
 # Using default() filter for hostvars 1st level key requires ansible 2.10+
 nginx_host: "{% if groups.nginx | length == 0 %}127.0.0.1{% else %}{{ hostvars[groups.nginx[0]].ansible_host | default('127.0.0.1') }}{% endif %}"
 repo_host: "{% if groups.repository | length == 0 %}127.0.0.1{% else %}{{ hostvars[groups.repository[0]].ansible_default_ipv4.address | default('127.0.0.1') }}{% endif %}"
-activemq_host: "{% if groups.activemq | length == 0 %}127.0.0.1{% else %}{{ hostvars[groups.activemq[0]].ansible_default_ipv4.address | default('127.0.0.1') }}{% endif %}"
+activemq_host: >-
+  {% if 'external_activemq' in groups %}
+  {{- groups['external_activemq'] | first -}}
+  {% else %}
+  {{- groups['activemq'] | map('extract', hostvars, ['ansible_default_ipv4','address']) | first | default('127.0.0.1') -}}
```

Comment:
I think this 127.0.0.1 default was only usefull for previous orles' tests which where using broken molecule inventory... We should try removing them as we see them.

Reply:
yeah, I think I need to update all the molecule configs to include the activemq task otherwise it will complain of missing address (since no hosts in activemq group)